### PR TITLE
DOC-2263: Set up staging 7 branch for upcoming docs release

### PR DESCRIPTION
**Pull request by MitchC1999**
Ticket: DOC-2263

Changes:
* Updated workflows to build 7 site in parallel as new latest
* 

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: .github/workflows/feature_7_docs.yml [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 jobs:
   build:
     name: Build Docs and Deploy
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14]
+        node-version: [16]
```
Comment (kemister85) [inline .github/workflows/feature_7_docs.yml:19]: is their a reason we dont use the current `stable` version of node?

File: .github/workflows/release_7_docs.yml [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 jobs:
   build:
     name: Build Docs and Deploy
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14]
+        node-version: [16]
```
Comment (kemister85) [inline .github/workflows/release_7_docs.yml:18]: is their a reason we dont use the current stable version of node?

File: .github/workflows/staging_7_docs.yml [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
 jobs:
   build:
     name: Build Docs and Deploy
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
 
     strategy:
       matrix:
-        node-version: [14]
+        node-version: [16]
```
Comment (kemister85) [inline .github/workflows/staging_7_docs.yml:18]: is their a reason we dont use the current stable version of node?

**Review by kemister85 — APPROVED** [review]
LGTM
- [x] should we remove all the `nav.adoc` links to the release notes in this setup? such as http://docs-feature-7-doc-2263.staging.tiny.cloud/docs/tinymce/7/release-notes/
- [x] should we delete the `changelog.adoc` content also?

File: .github/workflows/feature_7_docs.yml (hunk 1 above)
Comment (MitchC1999) [reply .github/workflows/feature_7_docs.yml:19]: I only updated due to build issues, where the required minimum version was 16. I can update these to the current stable if its necessary.

Comment (TheSpyder) [reply .github/workflows/feature_7_docs.yml:19]: If the current LTS works, might as well use it. These versions bit-rot with nobody assigned to maintain them.

File: modules/ROOT/pages/ui-localization.adoc [hunk 4]
```diff
@@ -8,7 +8,4 @@ include::partial$configuration/language.adoc[]
 
 include::partial$configuration/language_url.adoc[]
 
-////
-undocumented
-include::partial$configuration/language_load.adoc[]
-////
+include::partial$configuration/language_load.adoc[]
```
Comment (TheSpyder) [inline modules/ROOT/pages/ui-localization.adoc:11]: This seems to be live in 6, why is it showing up as added in 7 🤔

**Review by TheSpyder — APPROVED** [review]

File: modules/ROOT/pages/ui-localization.adoc (hunk 4 above)
Comment (kemister85) [reply modules/ROOT/pages/ui-localization.adoc:11]: yeah not sure, it was merged directly to `release/6` on the 20th dec https://github.com/tinymce/tinymce-docs/pull/3027/files

Comment (MitchC1999) [reply modules/ROOT/pages/ui-localization.adoc:11]: This branch was made from release/docs-6, but the staging/docs-7 branch was made from staging/docs-6 so it is missing the live changes.

**Comment by kemister85** [comment]
@MitchC1999 not sure if we decided to do it in this PR, but just mentioning bumping the version for `tinymce_live_demo_url:` in both `antora-playbook-staging.yml` and `antora-playbook-dev.yml`? not sure what production is pointing towards.

**Pull request merged**